src/openrct2/CmdlineSprite.cpp

```diff
@@ -643,12 +638,8 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
             }
         }
 
-        rct_g1_element spriteElement;
-        uint8_t* buffer;
-
-        int32_t bufferLength;
-        if (!sprite_file_import(
-                imagePath, x_offset, y_offset, false, false, &spriteElement, &buffer, &bufferLength, gSpriteMode))
+        auto importResult = sprite_file_import(imagePath, x_offset, y_offset, false, false, gSpriteMode);
+        if (!importResult)
```

Comment:
```suggestion
        if (importResult == std::nullopt)
```
I'd do the `!importResult` too, but for consistency, the rest of the code usually does explicit comparison to `nullopt`